indico/modules/events/contributions/templates/management/contributions.html

```diff
@@ -6,6 +6,24 @@
     {% trans %}Contributions{% endtrans %}
 {% endblock %}
 
+{% block page_actions %}
+    {%- if event.type == 'conference' -%}
+        <a id="contributions-settings-dropdown"
+           class="i-button inconspicuous icon-settings arrow js-dropdown"
+           data-toggle="dropdown"></a>
+        <ul class="dropdown">
+            <li>
+                <a data-ajax-dialog data-reload-after="customData" href="#"
```

Comment:
one data attribute per line and let's put it after the title so they are all together